### PR TITLE
Add information about testing environment

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,3 +184,17 @@ $this->app['db']->connection('couchbase')
...
+
+## hacking
+
+To run tests there are should be following buckets created on local Couchbase cluster:
+
+``` php
+$cluster = new CouchbaseCluster('couchbase://127.0.0.1');
+$clusterManager = $cluster->manager('Administrator', 'password');
+$clusterManager->createBucket('testing', ['bucketType' => 'couchbase', 'saslPassword' => '', 'flushEnabled' => true]);
+$clusterManager->createBucket('memcache-couch', ['bucketType' => 'memcached', 'saslPassword' => '', 'flushEnabled' => true]);
```
Comment (avsej) [inline README.md:196]: @ytake I'm not sure how you are initializing `couchbase-memcached`. You are specifying port 11255. Do you have standalone memcached process listening it?

Comment (ytake) [reply README.md:196]: I use a Proxy Port. 
(because 11211 is the default port for memcached(local))

Comment (avsej) [reply README.md:196]: @ytake why Couchbase-Laravel cares about plain memcached? It doesn't support all Couchbase features

Comment (ytake) [reply README.md:196]: I know, but due to my work... 😢

Comment (avsej) [reply README.md:196]: I have added a line about memcached on 11255 to readme

**Pull request merged**